contrib/python/src/python/pants/contrib/python/checks/checker/constant_logic.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import ast
+
+from pants.contrib.python.checks.checker.common import CheckstylePlugin
+
+
+class ConstantLogic(CheckstylePlugin):
+  """Check for constants provided to and/or which will disregard the other operand's value."""
```

Comment:
`constants provided to and/or which will disregard`

Doesn't quite parse for me...

Reply:
Fixed!